scripts/modal_runner.py

```
@@ -113,6 +113,20 @@ def _run_flashinfer_command(command: str) -> str:
 
+    if not os.path.exists("3rdparty/cccl/cub"):
+        print("=== Downloading CCCL ===")
+        subprocess.run(
+            [
+                "git",
+                "clone",
+                "--depth=1",
+                "--branch=v3.3.2",
+                "https://github.com/NVIDIA/cccl.git",
+                "3rdparty/cccl",
+            ],
```

Ruff (0.15.10)

[error] Starting a process with a partial executable path

(S607)